src/e2e-test/resources/testdata/BQMT_CSV_TEST.csv

```diff
@@ -0,0 +1,15001 @@
+zip_code,residential_ratio,business_ratio,other_ratio,total_ratio,tablename

```

Comment:
Do we really need this big file? We can also test using 10 records?

Reply:
Done.